include/itkMinimalStandardRandomVariateGenerator.h

```diff
@@ -72,16 +72,19 @@ class VkFFTBackend_EXPORT MinimalStandardRandomVariateGenerator : public RandomV
   itkNewMacro(Self);
 
   /** initialize with a simple IntegerType */
-  void Initialize(int randomSeed);
+  void
```

Comment:
Maybe we should apply clang-format to ITKModuleTemplate?

Reply:
I installed `.clang-format` in the root directory of this project by copying the version from ITK.

Reply:
I will update the module template.